tests/api/metrics/test_metrics_spans.py

```diff
@@ -22,25 +23,19 @@
 )
 
 
-# helper for testing span depth limit edge case
-def create_nested_spans_recursively(metrics: CLIMetrics, num_spans: int = 1) -> None:
-    @metrics.start_span("nested_span")
-    def create_span():
-        nonlocal num_spans
-        num_spans -= 1
+# helper for testing span limits
+def create_spans(metrics: CLIMetrics, width: int, depth: int):
+    counter = count()
```

Comment:
This is here in case the test fails then we will have something to work with in the logs